docs/code-snippets/free_shipping_customization.md

```diff
@@ -22,113 +22,101 @@ Add this code to your child theme's `functions.php`, or via a plugin that allows
 
 ## Code Snippets
 
-### Enabling or Disabling Free Shipping via Hooks
-
-You can hook into the `is_available` function of the free shipping method.
-
-```php
-return apply_filters( 'woocommerce_shipping_' . $this->id . '_is_available', $is_available );
-```
-
-This means you can use `add_filter()` on `woocommerce_shipping_free_shipping_is_available` and return `true` or `false`.
-
 ### How do I only show Free Shipping?
 
-The following snippet hides everything but `free_shipping`, if it's available and the customer's cart qualifies. 
+The following snippet will hide all other shipping methods if Free Shipping available for the customer.
```

Comment:
I think it would be easier to understand if we do the following change: 

```suggestion
The following snippet will hide all other shipping methods if Free Shipping is available for the customer.
```

Reply:
Thank you, I didn't realize I missed "is". I just committed the change from here.